src/Feeds.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * Pinterest for WooCommerce Feeds related helper methods
+ *
+ * @package     Pinterest_For_WooCommerce/Classes/
+ * @version     1.0.0
+ */
+
+namespace Automattic\WooCommerce\Pinterest;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class handling fetch methods for feed profiles.
+ */
+class Feeds {
+
+	/**
+	 * Get a specific merchant feed using the given arguments.
+	 *
+	 * @param string $merchant_id The merchant ID the feed belongs to.
+	 * @param string $feed_id     The ID of the feed.
+	 *
+	 * @return mixed
```

Comment:
It looks like we only have one possible return value that isn't an `Exception`. Should this be `object` instead of `mixed`?